src/functions/OverloadedOoops.kts

```diff
@@ -0,0 +1,29 @@
+package functions.overloadedOoops
+
+class Test {
+
+    fun test(block: () -> Int = { 0 }): String {
+        return "${block()}"
+    }
+
+    fun test2(block: () -> Int = { 0 }): () -> String {
+        return { test(block) }
+    }
+}
+
+
+{ println("Start") }()
```

Comment:
Why IIFE? never seen that in Kotlin before

Reply:
I mean I know, why now, just curious how you ended up with an IIFE in Kotlin

Reply:
@TWiStErRob Actually, I try to make lambda not to be alone here. That's why it looks so